### PR TITLE
Changes for Task 30

### PR DESCRIPTION
**Pull request by dulasevich**

### CODE REVIEW

File: src/test/java/YandexTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        login.sendKeys(LOGIN);
+        login.submit();
+
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(1));
+
+        WebElement password = driver.findElement(passwordField);
+        password.sendKeys(PASSWORD);
+        password.submit();
+
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(3));
```
Comment (LikBez123) [inline src/test/java/YandexTest.java]: implicit waiter is specified only once. Move this to @BeforeEach and remove 35 line

File: src/main/java/by/Variables.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.openqa.selenium.By;
+
+public class Variables {
+    public static By loginButton = By.linkText("Log in");
+    public static By loginField = By.name("login");
+    public static By passwordField = By.id("passp-field-passwd");
+    public static By emailsCheckbox = By.className("mail-Toolbar-Item-Checkbox");
+    public static By writeEmailButton = By.partialLinkText("Напи");
+    public static By forwardEmailIcon = By.xpath("//div[@data-key='view=toolbar-button-forward&threaded=yes&current_folder=1&sort_type=date&with_pins=yes&layout=messages&id=forward']");
```
Comment (LikBez123) [inline src/main/java/by/Variables.java]: in order not to write such a long and complex locator, you can use `contains()`. Ex.:
`//div[contains(@data-key, 'forward')]`
define the difference between `//div[contains(@data-key, 'forward')]` and `.//div[contains(@data-key, 'forward')]`

File: src/main/java/by/Variables.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    public static By pageTitle = By.tagName("title");
+    public static By currentDate = By.cssSelector("span[class^='Text Text_weight_bold ToggleWidget-DayOfMonth']");
```
Comment (LikBez123) [inline src/main/java/by/Variables.java]: there is no need to use the '^' sign if you use the full class name.
what is the diff between `class^=` , `class*=` and `class$=` ?

**Review by LikBez123 — APPROVED** [review]
fix comment

File: src/main/java/by/Variables.java (hunk 3 above)
Comment (dulasevich) [reply src/main/java/by/Variables.java]: as we discussed during the call ^ - for the beginning of class name, * - for the partial overlap in between the beginning and start, $ for the end of the class name

File: src/main/java/by/Variables.java (hunk 2 above)
Comment (dulasevich) [reply src/main/java/by/Variables.java]: I found an answer, but not sure I understand that - // will search for the elements throughout the whole document, while .// will start from the current position. However for our example the result is the same.

File: src/test/java/YandexTest.java (hunk 1 above)
Comment (dulasevich) [reply src/test/java/YandexTest.java]: updated

**Pull request merged**